src/vellum/workflows/types/definition.py

```diff
@@ -0,0 +1,39 @@
+from uuid import UUID
+from typing import Annotated, Any, Dict, Union
+
+from pydantic import BeforeValidator
+
+from vellum.client.types.code_resource_definition import CodeResourceDefinition as ClientCodeResourceDefinition
```

Comment:
In the future, I'd like to consolidate it down to just the type, but need to see:
- What is our appetite for extending the client types from client-generator
- Once we could start reliably generating UUID's from our schema